openfusion_ros:
  openfusion_ros:
    ros__parameters:
      append_poses_frequency: 10.0  # Frequency at which to append poses to the map
      pointcloud_frequency: 1.0 # Frequency at which to publish pointclouds
      pose_min_translation: 0.05  # Minimum translation to consider a new pose
      pose_min_rotation: 5.0  # Minimum rotation to consider a new pose
      parent_frame: "map" # The frame in which the map is defined
      child_frame: "camera" # The frame in which the camera is defined
      rgb_topic: "rgb"
      depth_topic: "depth"
      debug_images: false # Enable debug images
      topk: 10
      depth_max: 10.0
      logging.enabled: true # Enable logging
      logging.log_file: "openfusion.log"
